tools/src/main/scala/scala/scalanative/interflow/Interflow.scala

```diff
@@ -147,7 +156,12 @@ class Interflow(val mode: build.Mode, val is32BitPlatform: Boolean)(implicit
 object Interflow {
   def apply(config: build.Config, linked: linker.Result): Seq[Defn] = {
     val interflow =
-      new Interflow(config.mode, config.compilerConfig.is32BitPlatform)(linked)
+      new Interflow(
+        config.mode,
+        config.compilerConfig.is32BitPlatform,
+        config.compilerConfig.maxInlineDepth,
+        config.compilerConfig.maxCallerSize
```

Comment:
With so many arguments, maybe we can make a small refactor and pass config directly to the Interflow?